doc/sql.extensions/README.set_transaction.txt

```diff
@@ -27,6 +27,12 @@ commit fails. This option is mostly used by gfix.
 LOCK TIMEOUT nonneg_short_integer: it's the time (measured in seconds) that a
 transaction waits for a lock in a record before giving up and reporting an error.
 
+AUTO RELEASE TEMP BLOBID: makes the transaction release temporary ID of user BLOB
```

Comment:
May mention the tpb constant here too.

Reply:
I don't know if it's an appropriate place to mention TPB constants. But if it is, then it should be added for other options as well.

Reply:
We may decide this beyond the PR if no one minds.